### PR TITLE
Upgrade bedrock-quasar to v1.

### PR DESCRIPTION
**Pull request by aljones15**
- Remove q-field from q-input
- Add eslint-config-digitalbazaar to project
- Initial eslint setup.

Tests:
1. touch the legal name input then blur (should be invalid now and show red)
2. should be able to select an entityType
3. should be able to enter text into the input and input is valid.

Docs:
https://v1.quasar-framework.org/

please note this PR and other quasar v1 upgrades are all pending until the entire project has been updated and tested.

### CODE REVIEW

**Comment by aljones15** [comment]
p.s. the karma test from the test dir is broken. checked and this also appears to be the case in master.

**Comment by gannan08** [comment]
I think we should remove the use of filtering, other than that everything LGTM. Don't like the ergonomics of the filter and autocomplete feature.

**Review by gannan08 — CHANGES_REQUESTED** [review]
Remove the use of filtering/autocomplete until we find a generalized pattern that has a good user experience.

File: BrOrganizationForm.vue [hunk 1]
```diff
@@ -79,7 +76,8 @@ export default {
   },
   data() {
     return {
-      entityTypes
+      entityTypes,
+      previousEntityType: null
```
Comment (gannan08) [inline BrOrganizationForm.vue]: This value is unused. Delete from data.

**Review by gannan08 — CHANGES_REQUESTED** [review]

**Comment by aljones15** [comment]
@gannan08 also I can not get the q-select `@blur` and `@keyup` methods to fire while keyup is not surprising I would think that `@blur` would still work. Looking into this. 

q-select intercepts (does not forward):
```
@blur
@keyup (tried quite a few there might be some it does not see)
@keydown (just tried keydown no specific key)
@click
```
meaning the only way to really validate would be to call this.$v.$touch() on form submit or to use the new in built rules.

File: entityTypes.js [hunk 2]
```diff
@@ -30,5 +30,9 @@ export default [
   {
     "label": "Professional Corporation",
     "value": "PC"
+  },
+  {
```
Comment (gannan08) [inline entityTypes.js:34]: Why was this added?

**Review by gannan08 — CHANGES_REQUESTED** [review]
This needs to be rebased on the head of `update-company-types`.

**Comment by aljones15** [comment]
@gannan08 closing this I pushed a rebased branch with the quasar upgrade rebased from update-company types.

**Pull request closed without merging**